DQM/DTMonitorClient/src/DTDataIntegrityTest.cc

```diff
@@ -24,7 +23,7 @@
 using namespace std;
 using namespace edm;
 
-DTDataIntegrityTest::DTDataIntegrityTest(const ParameterSet& ps) : nevents(0) {
+DTDataIntegrityTest::DTDataIntegrityTest(const ParameterSet& ps) : nevents(0), mappingToken_(esConsumes()) {
```

Comment:
seems it should be 
DTDataIntegrityTest::DTDataIntegrityTest(const ParameterSet& ps) : nevents(0), mappingToken_(esConsumes<edm::Transition::EndLuminosityBlock>()) {